models.py

```diff
@@ -499,6 +503,20 @@ def validate_csv(data):
         rows.append((recipient, message, amount))
     return rows
 
+def format_amount(self, context, model, name):
+    amount = int2asset(model.amount)
```

Comment:
I want you to do the `X.XX` formatting in int2asset so all the other html/email templates that use it will be improved

Reply:
this should be changed as well.